cosmolog/cosmologger.py

```diff
@@ -208,9 +208,16 @@ def debug(self, *args, **kwargs):
         return self.log(logging.DEBUG, *args, **kwargs)
 
     def log(self, lvl, *args, **kwargs):
+        payload = kwargs.copy()
```

Comment:
any reason this should be deepcopy?

Reply:
a shallow copy is sufficient in this instance